### PR TITLE
Proposal for Lite using profile restrictions

### PR DESCRIPTION
**Pull request by goneall**
This is an initial proposal for how we may describe the Lite profile and serialization.

The profile describes the additional restrictions and the serialization describe the subset of supported fields.

This is just a draft at this point and does not include the complete set of classes and properties for the profile and the serialization.

### CODE REVIEW

**Comment by goneall** [comment]
This is an alternative to #521 and follows the proposal in issue #462

cc @yoshi-i @zvr

**Comment by goneall** [comment]
@zvr - does this match the expected format for describing profile conformance?

File: model/Lite/Lite.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Profile conformance
+
+For a /Software/Package to be conformant with this profile,
+the following has to hold:
+
+1. The mincount for /Core/Element/name is 1
+1. The mincount for /Software/Package/packageVersion is 1
+1. The mincount for /Software/Package/packageUrl is 1
+1. The mincount for /Software/SoftwareArtifact/copyrightText is 1
```
Comment (zvr) [inline model/Lite/Lite.md:32]: I think also `downloadLocation` and `homePage` , but I will let @yoshi-i comment on this.

**Review by zvr — APPROVED** [review]
The format looks fine to me.

File: model/Lite/Lite.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+For a /Software/Sbom to be conformant with this profile,
+the following has to hold:
+
+1. for every `/Software/Package` there MUST exist exactly one `/Core/Relationship`
+   of type `concludedLicense` having that element as its `from` property
+   and an `/SimpleLicensing/AnyLicenseInfo` as its `to` property.
+1. for every `/Software/Package` there MUST exist exactly one `/Core/Relationship`
+   of type `declaredLicense` having that element as its `from` property
```
Comment (kestewart) [inline model/Lite/Lite.md:41]: @zvr - shouldn't this be `concludedLicense`?

Comment (zvr) [reply model/Lite/Lite.md:41]: @kestewart the first bullet is about `concludedLicense`, this one also requires `declaredLicense`.

**Comment by yoshi-i** [comment]
I've committed updated Lite.md as PR #563 , from forked repository due to write permissions for  #this.

**Review by kestewart — APPROVED** [review]
The outstanding comment about downloadLocation and homePage, can be resolved later.    Have opened https://github.com/spdx/spdx-3-model/issues/584 to track.

**Pull request merged**